omnigibson/utils/bddl_utils.py

```diff
@@ -846,6 +897,10 @@ def _filter_object_scope(self, input_object_scope, conditions, condition_type):
                             # Reset if we're sampling a kinematic state
                             if condition.STATE_NAME in {"inside", "ontop", "under"}:
                                 kwargs["reset_before_sampling"] = True
+                            elif condition.STATE_NAME in {"attached"}:
+                                kwargs["bypass_alignment_checking"] = True
+                                kwargs["check_physics_stability"] = True
+                                kwargs["can_joint_break"] = False
```

Comment:
add TODO here to note that we need to revert this when possible in the future?

Reply:
why do we need to revert this?